### ISSUE TITLE
Add Account Information to aws_organizations_organization Resource

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#8579]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Referring to AWS Organizations Organization account information has been a highly requested  feature, mentioned in the original Organizations feature request and contributed in the form of various open data source pull requests. While this information will be surfaced in the future via a data source potentially similar to one of those implementations, this information can also be baked directly into the existing [`aws_organizations_organization` resource](https://www.terraform.io/docs/providers/aws/r/organizations_organization.html) which environments are likely utilizing already and would not require the usage of another data source.

Initial account information to be returned for each account by the resource:

* ARN
* Email
* ID
* Name

### Affected Resource

* `aws_organizations_organization` resource

### Potential Terraform Configuration

```hcl
# This configuration is written in Terraform 0.12 syntax and using outputs for example purposes
resource "aws_organizations_organization" "example" {}

output "account_arns" {
  value = aws_organizations_organization.example.accounts[*].arns
}

output "account_emails" {
  value = aws_organizations_organization.example.accounts[*].emails
}

output "account_ids" {
  value = aws_organizations_organization.example.accounts[*].id
}

output "account_names" {
  value = aws_organizations_organization.example.accounts[*].names
}
```

**NOTE:** Splat syntax to retrieve all account IDs as a list directly via this new `accounts` attribute (e.g. `aws_organizations_organization.example.accounts.*.id` or `aws_organizations_organization.example.accounts[*].id`) only works in Terraform 0.12 or later. It may be possible to workaround this configuration language limitation in Terraform 0.11 and prior using `count = "length(aws_organizations_organization.example.accounts)"` and `some_argument = "${lookup(aws_organizations_organization.example.accounts[count.index], "id")}"` for downstream resources, although the Terraform AWS Provider maintainers cannot provide further guidance on workarounds like these.

### References

* #571 
* #3604
* #4384
* #4688
* #6842

**Comment by bflad** [comment]
Pull request submitted: #8581

**Comment by bflad** [comment]
Support for this new attribute has been merged and will release with version 2.10.0 of the Terraform AWS Provider, likely later today. 👍

**Comment by bflad** [comment]
This has been released in [version 2.10.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2100-may-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
